.gitignore

```diff
@@ -1,3 +1,7 @@
 .DS_Store
 *.nix
 *.backup
+*.tar
+*.tar.gz
+*.zip
+/yarn.lock
```

Comment:
I know we had this file pushed by accident. But it feels weird ignoring a file that cannot be there.

I mean it's not a cache file or a local configuration file...

Reply:
I would argue nothing in the .gitignore file is supposed to be there anyway

Reply:
> But it feels weird ignoring a file that cannot be there.

My point is, it was there so it can be - even though I have no idea how it did end up there.

I agree this is weird.

Reply:
You get the `yarn.lock`  when you do yarn in the root folder (it’s supposed to be done in the front folder).
But even then you’ll notice it before doing your commit, unless it’s a merge commit with hundreds of files.

Reply:
In an ideal world, yes but then, #2660